applications/nrf5340_audio/src/bluetooth/Kconfig

```diff
@@ -356,22 +356,21 @@ endchoice
 
 config BT_AUDIO_BROADCAST_NAME
 	string "Broadcast name"
-	depends on TRANSPORT_BIS
 	default "NRF5340_BROADCASTER"
+	# TODO: Add back 'depends on TRANSPORT_BIS' once applications are ready
 	help
 	  Name of the broadcast; not the same as BT_DEVICE_NAME.
 
 config BT_AUDIO_BROADCAST_NAME_ALT
 	string "Alternative broadcast name"
-	depends on TRANSPORT_BIS
 	default "NRF5340_BROADCASTER_ALT"
+	# TODO: Add back 'depends on TRANSPORT_BIS' once applications are ready
 	help
 	  Alternative name of the broadcast.
 
 config BT_AUDIO_USE_BROADCAST_NAME_ALT
 	bool "Use the alternative broadcast name"
 	default n
-	depends on TRANSPORT_BIS
```

Comment:
Is there any reason for not adding the TODO here?

Reply:
No, it was just forgotten/overlooked. Will add it